src/icrar/leap-accelerate-cli/tests/E2EPerformanceTests.cc

```diff
@@ -98,7 +98,9 @@ namespace icrar
             }
             else if(impl == ComputeImplementation::cuda)
             {
+#ifdef CUDA_ENABLED
                 auto result = cuda::Calibrate(*ms, ToDirectionVector(directions), 0.0, false);
+#endif
```

Comment:
If you upgarde to gtest 1.10 apparently you can do this:

```c++
#else
    GTEST_SKIP_("Library built without CUDA support, skipping CUDA test");
#endif
```

The the rest of the `ifdef`s in this file are unnecessary, and you don't need to manually skip/disable tests.

Of course if you change to a runtime check then this would be slightly different, but the same idea.

Reply:
Only concern with updating gtest at the moment is that I need nvcc to able to compile gtest.h. I'll create an issue outlining the steps check while doing this.

Reply:
See https://github.com/ICRAR/leap-accelerate/issues/89